### ISSUE TITLE
CSS and JS files with URL parameters are not found

### ISSUE THREAD

**Issue by skycaptain** [vscode-restructuredtext/vscode-restructuredtext#279]
Please follow [the troubleshooting guide](https://docs.restructuredtext.net/articles/troubleshooting.html) to locate the culprit.

If troubleshooting does not reveal any useful information, provide the following information to create a new issue,

**Environment**
- [X] Extension version: v144.0.0
- [X] Visual Studio Code version: 
  Version: 1.53.2
  Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
  Date: 2021-02-11T11:45:54.515Z (1 wk ago)
  Electron: 11.2.1
  Chrome: 87.0.4280.141
  Node.js: 12.18.3
  V8: 8.7.220.31-electron.0
  OS: Darwin x64 20.3.0
- [ ] Operating System version:
- [ ] Python version:
- [ ] sphinx version: 
- [X] A sample of conf.py or its important fragments:

```python
html_theme = "furo"
```

**Expected behavior**

CSS and JS files are found and Sphinx preview is rendered properly formatted.

**Actual behavior**

CSS and JS files from [furo theme](https://github.com/pradyunsg/furo) are not found and hence Sphinx preview is rendered badly (not to say unusable).

**Detailed Steps to reproduce the problem**

Just use the [furo theme](https://github.com/pradyunsg/furo) on any project and enable Sphinx preview. In the Webview Developer Console, I got the following errors:

```plain
furo.css%3Fdigest%3D33d2fc4f3f180ec1ffc6524e273e21d7d58cbe49:1 Failed to load resource: the server responded with a status of 404 (Not Found)
furo-extensions.css%3Fdigest%3D26485485040e7aaf717c13fd0188a5ad2c2deb60:1 Failed to load resource: the server responded with a status of 404 (Not Found)
main.js%3Fdigest%3De931d09b2a40c1bb82b542effe772014573baf67:1 Failed to load resource: the server responded with a status of 404 (Not Found)
furo.css%3Fdigest%3D33d2fc4f3f180ec1ffc6524e273e21d7d58cbe49:1 Failed to load resource: the server responded with a status of 404 (Not Found)
furo-extensions.css%3Fdigest%3D26485485040e7aaf717c13fd0188a5ad2c2deb60:1 Failed to load resource: the server responded with a status of 404 (Not Found)
```

In their theme they append a hash parameter to the CSS and JS files (see [source](https://github.com/pradyunsg/furo/blob/32bac23251d7857cd81fa468b78eb77357b3fb53/src/furo/__init__.py#L115)). In RSTEngine URL parameters seem to be encoded and hence the theme's CSS and JS files are not found. I don't know what the best practice is and whether this issue should be reported here or in their project, but since the docs are rendered fine, when I open and preview the HTMLs from the build output even in VSCode, I would expect the same behaviour from this extension.

**Comment by lextm** [comment]
Fixed in release 146.0.0.

**Comment by skycaptain** [comment]
Confirmed. Thank you very much @lextm!
